### DISCUSSION TITLE
Editing wifi-settings via ssh possible?

### DISCUSSION THREAD

**Discussion by mrManniManfred** [rern/rAudio#542] [General]
Hi,
is it possible to edit the wifi-settings after the first boot via ssh login?

If so how?

I tried creating a wifi file in the boot directory and reboot. The file is gone after the boot but the wifi-settings are still wrong.

Currently my wifi is not connected but the SSID is shown correctly.

The GUI does not seem to work. It is not possible to forget the network or to manually set it. The buttons can be clicked and the dialogs open but after comfirming with the ok button there is not change.

I am trying to configure a static IP.

So is there a way to restore / reset the wifi settings by editing or deleting some files?

I am using the newest version of rAudio on a raspberry PI 3B.

**Comment by ceejayemm** [comment]
Why not recreate the 'wifi' file as you need it and reboot. The new 'wifi' file will be detected and the new settings applied.  This has worked for me in the past I have had similar problems.

Chris

**Comment by R101G** [comment]
@mrManniManfred 
You can check the settings in the SSID file in /etc/netctl. Make sure the interface name in the file is the same as is shown by the iwconfig command. File content should be something like this:

Interface=wlan0
Connection=wireless
ESSID="yourSSID"
IP=static
Security=wpa
Key="yourwifikey"
Hidden=no
Address='192.168.1.6/24'
Gateway='192.168.1.1'

**Comment by mrManniManfred** [comment]
Hi,
thank you both for your comments.

The information from the wifi file was loaded correctly but I entered a wrong gateway.

I was able to fix it in /etc/netctl :)...
